python/paddle/fluid/layers/utils.py

```diff
@@ -0,0 +1,54 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def convert_to_list(value, n, name):
```

Comment:
Maybe called "convert_to_int_list"? Since it only supports int types? Or make the function more general to other types?

Reply:
Great!  `convert_to_list ` can be written more generally, it can convert any numerical type to a list.